firebase-android-release-dashboard/src/components/ReleaseList/ReleaseList.js

```diff
@@ -40,14 +42,10 @@ function ReleaseList({releases}) {
         ))}
       </Grid>
       {releases.map((release) => (
-        <ReleaseCard key={release.releaseName} release={release} />
+        <ReleaseCard key={release.id} releaseId={release.id} />
```

Comment:
Are these params always the same? Is there an opportunity here to remove a param?

Reply:
These aren't necessarily duplicates since `key` is a special React prop that is used for updating components, and is not passed to the `ReleaseCard`. Since it's not passed to the component, we have to pass it with the extra `releaseId` param.